main.nf

```diff
@@ -330,7 +339,7 @@ Channel.from(summary.collect{ [it.key, it.value] })
  * Parse software version numbers
  */
 process get_software_versions {
-    publishDir "${params.outdir}/pipeline_info", mode: 'copy',
+    publishDir "${params.outdir}/pipeline_info", mode: params.publish_dir_mode,
```

Comment:
I guess we need to make this change for all other processes too - I'll do that in a separate PR to check some other stuff now though 👍🏻